packages/suite/src/support/messages.ts

```diff
@@ -6352,6 +6357,11 @@ const definedMessages = defineMessages({
         defaultMessage: 'After closing other tabs and windows, try refreshing this page.',
         id: 'TR_TROUBLESHOOTING_CLOSE_TABS_DESCRIPTION',
     },
+    TR_TROUBLESHOOTING_CLOSE_TABS_DESCRIPTION_DESKTOP: {
+        defaultMessage:
+            'After closing other browser tabs and windows, try quitting and reopening the Trezor Suite app.',
```

Comment:
The easier solution is to reload the app the same way the browser can be reloaded. However, it is discutable if we want to inform a user that he can reload the app using ctrl/cmd + R

Reply:
After discussion, we agreed to preserve the original variant.